Update to highlight a bit more the use-case (#16107)
DataDog/documentation: content/en/security_platform/application_security/setup_and_configure.md

## Track authenticated bad actors

> Most critical attacks are usually performed by authenticated users as they can access your most sensitive endpoints. To identify bad actors that are generating suspicious security activity, add user information to traces by instrumenting your services with the standardized user tags. You can add custom tags to your root span, or use instrumentation functions. Read [Tracking User Activity][1] for more information.

Reviewer: ```suggestion
Many critical attacks are performed by authenticated users who can access your most sensitive endpoints. To identify bad actors that are generating suspicious security activity, add user information to traces by instrumenting your services with the standardized user tags. You can add custom tags to your root span, or use instrumentation functions. Read [Tracking User Activity][1] for more information.
```
